pkgs/applications/networking/onionshare/fix-paths.patch

```diff
@@ -72,63 +35,6 @@ index 3373462..7fd245b 100644
 +        tor_geo_ip_file_path = "@geoip@/share/tor/geoip"
 +        tor_geo_ipv6_file_path = "@geoip@/share/tor/geoip6"
 +        obfs4proxy_file_path = "@obfs4@/bin/obfs4proxy"
-
+ 
```

Comment:
```suggestion

```

Reply:
That broke the build, patch format is like that